src/RandomCommand.php

```diff
@@ -33,5 +36,17 @@ public function handle()
         if (rand(0, 1000) === 42) {
             shell_exec('open https://en.wikipedia.org/wiki/Special:Random');
         }
+
```

Comment:
It's not immediately clear what this code does.

Please extract this to a function named `removeRandomVendorFile`